sdk/samples/iot/README.md

```diff
@@ -277,7 +283,7 @@ The following samples use SAS authentication to connect to Azure IoT Hub or Azur
 
 ## Next Steps and Additional Documentation
 
-Start using the IoT Provisioning Client in your solutions!
+Start using the IoT Clients in your solutions!
```

Comment:
Good catch!

Reply:
should we expand to Azure Embedded C SDK IoT Client   ?

Reply:
Looping @wduraes in here for some PM oracle guidance :)

Reply:
Love the additional clarity.